src/cls/IPM/Utils/Module.cls

```diff
@@ -1338,7 +1351,10 @@ ClassMethod LoadNewModule(
         if tVerbose || $get(pParams("Multicompile","Verbose"),0) {
             write !,"Loading "_tDisplayName_" in process "_$job
         }
-        set tSC = ##class(%IPM.Storage.Module).ExecutePhases(tModuleName,$listbuild("Activate"),1,.pParams)
+        // LoadNewModule() may be called on an install/load or an update. In the case of an update, invoke ApplyUpdateSteps lifecycle phase.
```

Comment:
Everything after the first sentence is unnecessary since its obvious in the code. Explaining the context of LoadNewModule (when it can get called) should be in the method description and not here

Reply:
In general, comments are good as long as they are explaining overarching parts of code or some slightly obscure looking code (like regex/$zstrip). If the code is clear, then comments get in the way and can easily get outdated (outdated comments are worse than no comments IMO)